src/main/java/uk/ac/cam/cl/dtg/segue/api/userAlerts/UserAlertsWebSocket.java

```diff
@@ -299,7 +300,10 @@ public void onClose(final Session session, final int status, final String reason
     @OnWebSocketError
     public void onError(final Session session, final Throwable error) {
         long connectedUserId = connectedUser.getId();
-        log.warn(String.format("Error in WebSocket for user (%s): %s", connectedUserId, error));
+        String errorString = error.toString();
+        if (!errorString.contains(IDLE_TIMEOUT_MESSAGE)) {
```

Comment:
I would feel better checking the Exception's type rather than the message but can't find one which is more specific than WebSocketTimeoutException or concurrent.TimeoutException which would hide more than just the idle case.
Not an important enough problem to worry too much about it being accidentally reintroduced on library upgrade.

Reply:
My thought process exactly! 😅